interpreter/astwalker.go

```diff
@@ -223,17 +219,17 @@ func (w *astWalker) walkCall(node *expr.Expr) []Instruction {
 		instructions = append(instructions, condition...)
 		// 1: jump to <END> on undefined/error
 		instructions = append(instructions,
-			NewJump(conditionId, len(trueVal)+len(falseVal) + 3,
+			NewJump(conditionId, len(trueVal)+len(falseVal)+3,
```

Comment:
Formatting is backsliding?  Is this the way gofmt wants it?

Reply:
Hmm, this must be what go fmt wants.